<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Układ równań - metoda Jacobiego</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calcBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Oblicz</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rozwiązywanie układu równań liniowych metodą Jacobiego</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtn1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arytmetyka zwykła</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtn2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arytmetyka zmiennopozycyjna</string>
    </property>
   </widget>
   <widget class="QLabel" name="nLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Podaj wielkość macierzy (n):</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorLbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Źle wprowadzone dane!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTxtBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>331</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="wynikLbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wyniki obliczeń:</string>
    </property>
   </widget>
   <widget class="QLabel" name="error2Lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(Nie możesz obliczyć dopóki nie podasz wszystkich wartości)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtn3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Arytmetyka przedziałowa zmiennopozycyjna</string>
    </property>
   </widget>
   <widget class="QLabel" name="error3Lbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wartości na przekątnej muszą być różne od 0!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nTxtBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="dataTxtBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="creatorLbl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Stworzone przez Remigiusza Janickiego</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wpisując liczbę, znak &quot;.&quot; to oddzielenie całości od części ułamkowej</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Znak &quot;,&quot; odpowiada natomiast za oddzielenie lewego przedziału od prawego</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Przycisk &quot;ENTER&quot; na klawiaturze spowoduje zatwierdzenie liczby o ile jest poprawnie wpisana</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionv">
   <property name="text">
    <string>v</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
